gnovm/cmd/gno/testdata/gno_test/fmt_empty.txtar

```diff
@@ -0,0 +1,10 @@
+gno fmt noimport.gno
+cmp stdout stdout.golden.gno
+cmp stderr stderr.golden
+
+-- noimport.gno --
+package hello
+-- stdout.golden.gno --
+package hello
+
+-- stderr.golden --
```

Comment:
`testdata/gno_test` was originally created for the `gno test` command test cases. It seems to have strayed from its purpose, as it contains other command test cases such as `gno lint` and now `gno fmt` (probably because the name is confusing). Just as there are `testdata/gno_transpile` and `testdata/gno_precompile` directories, there should be a `testdata/gno_fmt` and a `testdata/gno_lint` directory.

Reply:
Nice catch. I've split fmt and lint as you suggested. I was thinking it was meant for all Txtar tests, generally speaking, since at the time there was only a single directory. Having multiple directories will make it clear that you need to split those tests.